lib/manticore/client.rb

```diff
@@ -618,7 +620,7 @@ def ssl_socket_factory_from_options(ssl_options)
         raise "Invalid value for :verify. Valid values are (:all, :browser, :default)"
       end
 
-      context = SSLContexts.custom
+      context = SSLContextBuilder.new
```

Comment:
[equivalent to what the `.custom` did](https://github.com/apache/httpcomponents-client/blob/4.5.x/httpclient/src/main/java-deprecated/org/apache/http/conn/ssl/SSLContexts.java#L91) - one less (deprecated) class to worry about